maro/simulator/scenarios/vm_scheduling/virtual_machine.py

```diff
@@ -51,6 +55,9 @@ def __init__(
         self.creation_tick: int = -1
         self.deletion_tick: int = -1
 
+    def get_income_till_now(self, cur_tick: int):
+        return self.unit_price * (cur_tick - self.creation_tick)
```

Comment:
add docstring to illustrate that the income got here also contains income from [creation_tick, current_tick)